debian/changelog

```diff
@@ -707,7 +707,7 @@ ubuntu-advantage-tools (27.11~22.10.1) kinetic; urgency=medium
     - pro:
       + Ubuntu Pro is released as a product
       + make `pro` the recommended executable for the client
-      + client, apt and motd messages updated/rewritten to show Pro
+      + client, apt and motd messages updated/rewritten to show Pro 
```

Comment:
Since this is up for review... Extra whitespace at the end of the line? ;)

Reply:
@panlinux this is in response to your comment here :) https://github.com/canonical/ubuntu-pro-client/pull/3247#discussion_r1739416521

Reply:
Ah, you are undoing that change, ok, I should diff against pkg/ubuntu/devel

Reply:
Did the diff against pkg/ubuntu/devel, no noise, just the new d/changelog entry, all good.